### DISCUSSION TITLE
No access to Github Copilot

### DISCUSSION THREAD

**Discussion by dudduss** [community/community#40028] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

I keep seeing this error in VSCode.

```
no access to github copilot found
```

I just recently bought the annual subscription, but it stopped working all of a sudden. I've signed out of GH (on desktop and VSCode) and tried to get it working again but it won't.

**Comment by PabloLION** [comment]
The server is down.. see (my) discussion https://github.com/orgs/community/discussions/40026

**Comment by jfullstackdev** [comment, marked as answer]
I'm seeing a lot of these things, you can check this out since one GitHub Staff already responded there

https://github.com/orgs/community/discussions/40037
